types/aws-lambda/index.d.ts

```diff
@@ -34,6 +34,7 @@
 //                 Marian Zange <https://github.com/marianzange>
 //                 Alexander Pepper <https://github.com/apepper>
 //                 Alessandro Palumbo <https://github.com/apalumbo>
+//                 Sachin Shekhar <https://github.com/SachinShekhar>
```

Comment:
Be aware this will make you be notified of all PRs modifying this package.

Reply:
I know. That's the reason I added that line.

Reply:
👍 welcome to the team then!